### ISSUE TITLE
"Unsupported engine" warning when using Node 18

### ISSUE THREAD

**Issue by NovemLinguae** [wikimedia/jscs-preset-wikimedia#510]
In my package.json:

```js
  "devDependencies": {
    "eslint": "^8.43.0",
    "eslint-config-wikimedia": "^0.25.1",
    "eslint-plugin-jsdoc": "^46.2.6"
  },
```

Looking at the dependencies in package-lock.json, I suspect eslint-config-wikimedia is the problem, since it appears to have as dependencies `eslint-plugin-jsdoc` and `@es-joy/jsdoccomment@0.23.6`

<img width="331" alt="image" src="https://github.com/wikimedia/eslint-config-wikimedia/assets/79697282/7472d937-a6fd-4279-8ee7-0c340080052e">

**Comment by siddharthvp** [comment]
Should be fixed by #482

**Comment by Krinkle** [comment]
The upstream fix for this landed in eslint-plugin-jsdoc **39.2.5** (https://github.com/gajus/eslint-plugin-jsdoc/commit/21233bc4df3632bc6954c50d5e78b121a6183542). We were on 38.x when this issue was filed, but since https://github.com/wikimedia/eslint-config-wikimedia/pull/484 we are on **39.2.2**. Quoting @edg2s in that pull request:

> **39.2.3** updates some library that causes all our tests to fail - needs more investigation, but this should resolve https://github.com/wikimedia/eslint-config-wikimedia/issues/510
